README.md

```diff
@@ -1,100 +0,0 @@
-## DGL Implementation of the MixHopGCN Paper
-
-This DGL example implements the GNN model proposed in the paper [MixHop: Higher-Order Graph Convolution Architectures via Sparsified Neighborhood Mixing](https://arxiv.org/abs/1905.00067). The author's codes of implementation is in [here](https://github.com/samihaija/mixhop).
-
-## Example implementor
-
-This example was implemented by [xnuohz](https://github.com/xnuohz) during his intern at the AWS Shanghai AI Lab.
-
-### Requirements
-The codebase is implemented in Python 3.6.11. package versions used for devalopment are just below.
```

Comment:
"package versions used for devalopment are just below." -> "For version requirement of packages, see below."

Reply:
Done.